agimus_pytroller_py/agimus_pytroller_base.py

```diff
@@ -8,9 +8,45 @@
 
 
 class ControllerImplBase(ABC):
-    @abstractmethod
-    def __init__(self, robot_description: str) -> None:
+    def __init__(
```

Comment:
It's a personal opinion but:
1. I like default constructors as much as possible.
2. You are parsing something that is extremely mysterious and not indicative in what it should contain

Again personal opinion no need to take care of it here.
I would have created a parser somewhere that spits out a dataclass but that is me.
I understand that the dataclass is rigid an you might prefer a dictionary and if we go that way you might just parse the ROS message directly.